### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by michellepineda90**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,24 +1,21 @@
...
+import Panel from './components/Panel/Panel';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
+      <NavBar />
+      <Main />
+      <Panel />
+      {/* <header className="App-header">
```
Comment (josectobar) [inline src/App.tsx:13]: Delete commented code

File: src/components/CircleDetail/CircleDetail.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        return (
+            <div className='circles'>
+                <div className='circle floor'></div>
+                <div className='circle middle'></div>
+                <div className='circle withText ceil'>
+                    <p className='plusFiles'>+3</p>
+                </div>
+            </div>
+        )
+    }
```
Comment (josectobar) [inline src/components/CircleDetail/CircleDetail.tsx:25]: You should avoid leaving empty JSX components (`<div></div>`) and instead add then as self closed (`<div />`)
```suggestion
        return (
            <div className='circles'>
                <div className='circle floor'/>
                <div className='circle middle'/>
                {files < 3 && (
                  <div className='circle withText ceil'>
                    <p className='plusFiles'>+3</p>
                </div>
                )}
            </div>
        )
```

File: src/components/FileCard/FileCard.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    return (
+        <div className='fileCard'>
+            <div className={`smallSquare ${color}`}></div>
+            <p className='title'>{title}</p>
+            <p className='members txtColor'>{`${members} members`}</p>
+            <p className='fileTxt txtColor'>{date}</p>
+            <div className='ellipsis horizontal'>
+                <div className='dot'></div>
+                <div className='dot'></div> 
+                <div className='dot'></div>
```
Comment (josectobar) [inline src/components/FileCard/FileCard.tsx:20]:
```suggestion
               <div className='dot'/>
                <div className='dot'/>
                <div className='dot'/>
```

File: src/components/Gradient/Gradient.tsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+            <div className='gradient gradientOne'></div>
+        )
+    } else if( className === 'gradientTwo'){
+        return (
+            <div className='gradient gradientTwo'></div>
+        )
+    } else {
+        return null
+    }
+}
```
Comment (josectobar) [inline src/components/Gradient/Gradient.tsx:20]:
```suggestion
     if (!className) return null
    return className === 'gradientOne' ? (
            <div className='gradient gradientOne' />
        ) : (
            <div className='gradient gradientTwo' />
        )
    } 
```

File: src/components/NameSection/NameSection.tsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+function NameSection({name} : IProfile) {  
+    return (
+        <div className='nameSection'>
+            <div className='btns'>
+                <input type='button' className='btn btnPanel' />
+                <input type='button' className='btn btnPanel' />
+            </div>
+            <div className='name'>
+                <p className='regularTxt'>{name}</p>
+                <div className='profilePic'></div>
```
Comment (josectobar) [inline src/components/NameSection/NameSection.tsx:16]:
```suggestion
 <div className='profilePic' />
```

File: src/components/NavBarSections/NavBarSections.tsx [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    state: string;
+}
+
+function NavBarSections({name, state}: INavBarSection) {
+
+//   const [isActive, setIsActive] = useState(false);
+//   const handleClick = event => {
+//     setIsActive(current => !current);
+//     isActive class
+//   };
```
Comment (josectobar) [inline src/components/NavBarSections/NavBarSections.tsx:15]:
```suggestion
```

File: src/components/NavBarSections/NavBarSections.tsx [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+    function handleClick(event: { currentTarget: { classList: { toggle: (arg0: string) => any; }; }; }) {
+        return event.currentTarget.classList.toggle('selectedButton');
+    }
+
+    
+    if (state === 'isActive') {
+            return (
+                <div className='sectionItem'>
+                    <div className='pointer selected'></div>
```
Comment (josectobar) [inline src/components/NavBarSections/NavBarSections.tsx:25]:
```suggestion
                    <div className='pointer selected'/>
```

File: src/components/NavBarSections/NavBarSections.tsx [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+                    <input type='button' className='navBarBtn btn selectedButton' id={name} name={name} onClick={handleClick}/>
+                    <label htmlFor={name} className='label selectedLabel'>{name}</label>
+                </div>
+        )
+        } else {
+            return (
+                <div className='sectionItem'>
+                    <div className='pointer'></div>
```
Comment (josectobar) [inline src/components/NavBarSections/NavBarSections.tsx:33]:
```suggestion
                    <div className='pointer' />
```

File: src/components/Panel/Panel.tsx [hunk 9]
```diff
@@ -0,0 +1,22 @@
+import './Panel.css';
+import NameSection from '../NameSection/NameSection';
+import StorageSection from '../StorageSection/StorageSection';
+import BuySpaceSection from '../BuySpaceSection/BuySpaceSection';
+
+function Panel() {
+    return (
+        <div className='panel'>
+            <NameSection name='Michelle' />
+            <hr className='lineTwo'></hr>
```
Comment (josectobar) [inline src/components/Panel/Panel.tsx:10]:
```suggestion
            <hr className='lineTwo' />
```

File: src/components/RecentProjectsCard/RecentProjectsCard.tsx [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+    files: number;
+}
+
+function RecentProjectsCard({name, date, files}: IRecentCard) {
+    return (
+        <div className='card'>
+            <div className='cardIcons'>
+                <div className='cube'>
+                    <div className='square bottom'></div>
+                    <div className='square superposed'></div>
```
Comment (josectobar) [inline src/components/RecentProjectsCard/RecentProjectsCard.tsx:16]:
```suggestion
                    <div className='square bottom' />
                    <div className='square superposed' />
```

File: src/components/RecentProjectsCard/RecentProjectsCard.tsx [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+                </div>
+                <CircleDetail files={files} />
+                <div className='ellipsis vertical'>
+                    <div className='dot'></div>
+                    <div className='dot'></div>
+                    <div className='dot'></div>
```
Comment (josectobar) [inline src/components/RecentProjectsCard/RecentProjectsCard.tsx:22]:
```suggestion
                    <div className='dot' />
                    <div className='dot' />
                    <div className='dot' />
```

File: src/components/ShareCircleDetail/ShareCircleDetail.tsx [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+        )
+    } else {
+        return (
+            <div className='circles'>
+                <div className='circle aquaBorder one'></div>
+                <div className='circle magentaBorder two'></div>
+                <div className='circle orangeBorder three'></div>
+            </div>
+        )
+    }
```
Comment (josectobar) [inline src/components/ShareCircleDetail/ShareCircleDetail.tsx:23]:
```suggestion
    return (
            <div className='circles'>
                <div className='circle aquaBorder one' />
                <div className='circle magentaBorder two' />
                {files > 2 && (<div className='circle orangeBorder three' />)}
            </div>
    )
```

File: src/components/Storage/Storage.tsx [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+        { title: 'Documents', files: 720, gigabytes: 125, color: 'blue' },
+        { title: 'Documents', files: 720, gigabytes: 75, color: 'aqua' },
+        { title: 'Documents', files: 720, gigabytes: 50, color: 'gray'},
+    ]  
+
+    return (
+        <div className='storage'>
+            {files.map((file) => (
+                <StorageDetail
+                    title={file.title}
```
Comment (josectobar) [inline src/components/Storage/Storage.tsx:16]:
```suggestion
                    key={file.color}
                    title={file.title}
```

**Review by josectobar — COMMENTED** [review]
Good job @michellepineda90, I liked that you used variables with CSS. I left some comments.

File: src/App.tsx (hunk 1 above)
Comment (michellepineda90) [reply src/App.tsx:13]: Totally, thanks for noticing 😅 !

File: src/components/CircleDetail/CircleDetail.tsx (hunk 2 above)
Comment (michellepineda90) [reply src/components/CircleDetail/CircleDetail.tsx:25]: True. I don't believe I added a linter to the project, but now I am aware of this convention. Thanks!

**Comment by michellepineda90** [comment]
Thank you for your feedback, @josectobar 🥇 !

**Pull request merged**